### DISCUSSION TITLE
How to make sqladmin request static files (css, js, etc.) via https, not http?

### DISCUSSION THREAD

**Discussion by iQiexie** [aminalaee/sqladmin#596] [Q&A] [answered]
The problem is that my application is running begind nginx proxy that redirects all http requests to https and because of that styles for sqladmin are not loading

**Comment by aminalaee** [comment, marked as answer]
Hey, check this out: https://aminalaee.dev/sqladmin/cookbook/deployment_with_https/

**Reply by AidarHaziahmetov** [reply]
Not allowed

**Reply by eslam5464** [reply]
https://aminalaee.dev/sqladmin/cookbook/deployment_with_https/

![image](https://github.com/user-attachments/assets/c0c02ddd-bf44-4f3d-880e-690604cd210a)


The above url is not working anymore, you can use the below url

https://aminalaee.github.io/sqladmin/cookbook/deployment_with_https/

**Comment by AidarHaziahmetov** [comment]
Thank you so much

пт, 6 июн. 2025 г., 21:38 Eslam ***@***.***>:

**Reply by eslam5464** [reply]
You're most welcome
